README.md

```diff
@@ -16,6 +16,23 @@ The `rlistings` R package is a package that was designed to create and display l
 
 `rlistings` currently covers some of these requirements, and remains under active development.
 
+## Simple example
+
+The following example shows a simple listings and its printed format.
+
+```{r}
+library(dplyr)
+library(rlistings)
+
+# Reducing the data
+mtcars_ex <- mtcars %>% mutate("car" = rownames(mtcars)) %>% head(10)
+
+as_listing(mtcars_ex, 
+  key_cols = c("gear", "carb"), 
+  cols = c("gear", "carb", "qsec", "car"))
+  
+```
+
```

Comment:
@Melkiades @edelarua if we're going to have code and output in the README.md file (which I think is a good idea), it needs to be generated via a README.Rmd file (which is how the output will actually get in there). Can one of you please make this change?

Reply:
I have added an README.Rmd file and regenerated the README.md accordingly.